### PR TITLE
added KDE Plasma 5 support

### PR DESCRIPTION
**Pull request by piggyslasher**
From https://github.com/zealdocs/zeal/issues/471#issuecomment-172138023

### CODE REVIEW

File: lib/dash.coffee [hunk 1]
```diff
@@ -86,7 +86,10 @@ plugin = module.exports =
     return keys.map(encodeURIComponent).join(',') if keys.length
 
   getDashURI: (string, path, language, background) ->
-    link = 'dash-plugin://query=' + encodeURIComponent(string)
+    if process.env.XDG_SESSION_DESKTOP == "KDE"
+      link = 'dash-plugin:query=' + encodeURIComponent(string) #plays nice with KDE Plasma 5
+    else link = 'dash-plugin//:query=' + encodeURIComponent(string)
```
Comment (blakeembrey) [inline lib/dash.coffee]: The typo here is breaking the build. It should be `://`.

Comment (blakeembrey) [inline lib/dash.coffee]: If you can place it on a new line too, that would be good.

Comment (trollixx) [inline lib/dash.coffee]: I wouldn't rush on this one. I have no idea why KDE doesn't like host-less
URIs, so I'd like to double check that, and perhaps talk to KDE people.

Comment (blakeembrey) [inline lib/dash.coffee]: Fair enough, if there's anyone you know who can check that'd be great.

Comment (trollixx) [inline lib/dash.coffee]: Just for the reference, my [comment](https://github.com/blakeembrey/atom-dash/issues/37#issuecomment-237140968) in #37 explains why removing double slash is a good idea for any Linux, not just KDE.

Comment (blakeembrey) [inline lib/dash.coffee]: No worries. I can do that. I'm going to close this PR though, it hasn't been updated since pointing out this typo anyway.

**Pull request closed without merging**